src/sidebar/components/test/MarkdownEditor-test.js

```diff
@@ -534,11 +535,13 @@ describe('MarkdownEditor', () => {
     it('allows changing highlighted suggestion via vertical arrow keys', () => {
       const wrapper = createComponent({
         mentionsEnabled: true,
-        usersForMentions: [
-          { username: 'one', displayName: 'johndoe' },
-          { username: 'two', displayName: 'johndoe' },
-          { username: 'three', displayName: 'johndoe' },
-        ],
+        usersForMentions: {
+          users: [
```

Comment:
There is a missing `status` prop here. The code happens to work without it, but this could break in future.

Reply:
It actually defaults to `{ status: 'loaded', users: [] }`, and then it`s extended with provided value:

https://github.com/hypothesis/client/pull/6862/files#diff-610127f61285415917a6c61f61bf88cb133bc08a24769ee8ac80e45f4576335fR60

I'll change it so that it's explicitly passed every time

Reply:
Ah, OK. If you want to save some duplication, creating a helper function with type `(users: UserItem[]) => UsersForMentions` would be an alternative.